src/Data/Nat/DivMod.agda

```diff
@@ -239,13 +240,13 @@ m≥n⇒m/n>0 {m@(suc _)} {n@(suc _)} m≥n = begin
   m / n ∎
 
 m/n≡0⇒m<n : ∀ {m n} .{{_ : NonZero n}} → m / n ≡ 0 → m < n
-m/n≡0⇒m<n {m} {n@(suc _)} m/n≡0  with <-≤-connex m n
+m/n≡0⇒m<n {m} {n} m/n≡0  with <-≤-connex m n
 ... | inj₁ m<n = m<n
 ... | inj₂ n≤m = contradiction m/n≡0 (≢-nonZero⁻¹ _)
   where instance _ =  >-nonZero (m≥n⇒m/n>0 n≤m)
 
 m/n≢0⇒n≤m : ∀ {m n} .{{_ : NonZero n}} → m / n ≢ 0 → n ≤ m
```

Comment:
I seem to be missing something here. 1 / 2 is not zero, so 2 ≤ 1 ???

Reply:
This is division of naturals, rounded towards zero. By this, `1 / 2` is in fact 0.

Reply:
Thanks, I was mis-understanding that `/` was.